src/Tmds.Systemd.Logging/JournalLoggerOptions.cs

```diff
@@ -1,13 +1,44 @@
+using System;
+
 namespace Tmds.Systemd.Logging
 {
     /// <summary>
     /// Options for the journal logger.
     /// </summary>
     public class JournalLoggerOptions
     {
-        /// <summary>Drop messages instead of blocking.</summary>
+        /// <summary>
+        /// Gets or sets a value indicating whether messages are dropped when busy instead of blocking.
+        /// </summary>
         public bool DropWhenBusy { get; set; }
-        /// <summary>The syslog identifier added to each log message.</summary>
+
+        /// <summary>
+        /// Gets or sets the syslog identifier added to each log message.
+        /// </summary>
         public string SyslogIdentifier { get; set; } = Journal.SyslogIdentifier;
+
+        /// <summary>
+        /// Gets or sets a delegate that formats an exception and sets message fields.
+        /// If unset, the default fields are set.
+        /// </summary>
+        public Action<Exception, JournalMessage> ExceptionFormatter { get; set; }
```

Comment:
Default this property to `DefaultExceptionFormatter`.

Reply:
It sounds like you have a clear idea of how things should look, and I'm having a hard time following these explanations. Maybe it would be easier if you describe the code changes in a commit instead?